.docker/browser/libero_page.yaml

```diff
@@ -3,6 +3,8 @@ libero_page:
         homepage:
             path: '/'
             primary_listing: 'search'
+        header_image:
+            src: /assets/homepage.jpg
```

Comment:
Not including variants to avoid having to commit multiple images. Could host them externally, or have a build process here that compiles them from the base image.

Reply:
assumption that this is included in the image (and it ends up in the shared volume)

Reply:
ah no, it's configured here

Reply:
the `src` path is the URL path rather than an absolute file location, right